README.md

```diff
@@ -1,4 +1,6 @@
-<p align="center"> <img alt="Space Station 14" width="880" height="300" src="https://raw.githubusercontent.com/space-wizards/asset-dump/de329a7898bb716b9d5ba9a0cd07f38e61f1ed05/github-logo.svg" /></p>
+<div class="header" align="center">  
```

Comment:
Not a change request, just a side-note: github "rich diff" breaks with centering here. "View file" actually shows that it is centered though, yay.